settings.py

```diff
@@ -16,6 +16,7 @@ class Settings(BaseSettings):
     use_x_forwarded_proto: bool = False
 
     postgres_dsn: PostgresDsn
+    postgres_connect_args: dict = {"connect_args": {"application_name": "api-reflector"}}
```

Comment:
may i suggest either calling this `postgres_engine_options` since that's what's actually being set here with connect args inside of it, or alternatively unwrapping the connect args like this:

```python
postgres_connect_args: dict = {"application_name": "api-reflector"}

# then in api.py
SQLALCHEMY_ENGINE_OPTIONS={"connect_args": settings.postgres_connect_args}
```

the first option allows us to set any engine option, the latter only allows setting connect_args. i'm fine with either really.

this should avoid any confusion about how the setting works.

Reply:
@thenu97 - make it so.